tests/client/core/test_middleware.js

```diff
@@ -0,0 +1,93 @@
+import { prefixMiddleWare } from 'core/middleware';
+
+describe('Prefix Middleware', () => {
+  let fakeRes;
+  let fakeNext;
+  let fakeConfig;
```

Comment:
I'm not too crazy about this pattern anymore. It's too easy to make a mistake and forget to reset one of the vars. I prefer doing this at the top of each test:

    const { fakeRes, fakeNext, fakeConfig } = getFakes();

It's one extra line of code but it's more explicit and less error prone.

Anyway, yours is fine as is but just wanted to throw that out there.